storage/pipeline/piece/piece_info.go

```diff
@@ -24,7 +24,7 @@ type PieceDealInfo struct {
 	DealID       abi.DealID
 	DealProposal *market.DealProposal
 
-	// Common deal info
+	// Common deal info, required for all pieces
 	DealSchedule DealSchedule
```

Comment:
@magik6k wont it be easier if this is passed in as a part of the PAM?

Reply:
Not all data pieces will have a PAM